cypress/integration/tests/controls/stakeholders/create.test.ts

```diff
@@ -30,6 +30,6 @@ describe("Create New Stakeholder", () => {
         cy.wait("@getStakeholders");
 
         // Assert that newly created stakeholder is deleted
-        cy.get(tdTag).should("not.contain", stakeholder.stakeholderEmail);
+        cy.get(tdTag).should("not.contain", stakeholder.getCurrentStakeholderName);
```

Comment:
why has this been changed? stakeholder email is the primary key, which is constant after edit. Please revert this change.